crates/core/src/factories/core_fetch/test.rs

```diff
@@ -139,6 +147,12 @@ mod tests {
             .await
             .unwrap();
 
```

Comment:
Required now to do the blocks check when sending messages, otherwise messages get dropped.